tests/testCountInputs.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+# Copyright 2008-2015 LSST/AURA
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+"""
+Tests for CountInputs measurement algorithm
+"""
+
+import unittest
+import lsst.utils.tests as utilsTests
+
+import lsst.pex.config as pexConfig
+import lsst.afw.geom as afwGeom
+import lsst.afw.table as afwTable
+import lsst.afw.image as afwImage
+import lsst.afw.coord as afwCoord
+import lsst.afw.detection as afwDetection
+import lsst.meas.base as measBase
+from lsst.afw.geom.polygon import Polygon
+
+def ccdVennDiagram(exp, showImage=True, legendLocation='best'):
```

Comment:
Is this really a Venn diagram? I think it just looks like one. That's a trivial point, though.

Is it possible to create this diagram using the standard `afw.display` code rather than custom matplotlib? That would be more consistent with the rest of the stack. I don't think you should spent significant effort on it, since it's not a fundamental part of the test, though.